ocs_ci/templates/cnv-vm-workload/vm_centos.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: centos-stream8
+  namespace: default
+
+spec:
+  dataVolumeTemplates:
```

Comment:
I noticed you are using datavolume templates which won't be supported in 4.15.0. For 4.15.0 we need the stand alone PVC which is referenced by the vm

An example of the stand alone PVC referenced by the vm can be seen here:
https://github.com/kgoldbla/ocm-kubevirt-samples/blob/vm9-application/vm-standalone-pvc-odr-regional/pvc.yaml

Reply:
Done